### DISCUSSION TITLE
Can we contact Elon Musk and suggest supporting the ActivityPub protocol for Twitter?

### DISCUSSION THREAD

**Discussion by MirceaKitsune** [mastodon/mastodon#18214] [General]
I know folks have divided opinions on Elon Musk buying Twitter, not interested in the politics of that here. The part I find relevant for this discussion is his stance on making the platform more free and modern, particularly discussion about potentially "making the Twitter algorithm open" which got me thinking. While rich and / or popular people are difficult to get a hold of I still imagined the following scenario: Could the Mastodon team get a hold of Elon, introduce him to the ActivityPub protocol if he isn't familiar already, and suggest he supports it for Twitter?

Like I said I know Twitter is often a dumpster fire so there are legitimate questions on the negative side effects such a move could have. If Twitter users can interact with people on Mastodon however, this could a huge win for a more connected internet and help different people communicate. Could it be attempted, should it, think it could ever happen in reality?

**Comment by markcellus** [comment]
Don't quite understand why you're being downvoted. Seems like a legitimate question. 

Can someone who thinks this idea isn't so great explain why? Very interested in seeing another perspective. 👍

**Reply by connyduck** [reply]
One of the main points is Twitter having problematic moderation policies, e.g. Terfs and Trolls largely go unchecked. With Twitter in Fedi you get all that toxic behavior like dogpiling into it as well. A small instance cannot control that sudden influx of hate, probably not even mastodon.social can. They'll all defederate. There is a reason people are in the Fedi and not on Twitter. We don't want to connect with it.

**Reply by MirceaKitsune** [reply]
Instances who are concerned about that could easily blacklist Twitter as they can any other server. Those who are larger and have more resources or simply feel adventurous can federate with it, which would be interesting to see at such a scale for the first time I think. It would also help to have externally maintained block lists for accounts known to be problematic, users could install those and automatically be rid of what they don't want to see... that's the way I always thought moderation needs to be done rather than universally by central authority.
